k256/src/arithmetic/scalar.rs

```diff
@@ -120,8 +120,8 @@ impl Scalar {
     /// Right shifts the scalar.
     ///
     /// Note: not constant-time with respect to the `shift` parameter.
-    pub fn shr_vartime(&self, shift: usize) -> Scalar {
-        Self(self.0.shr_vartime(shift))
+    pub fn shr_vartime(&self, shift: u32) -> Scalar {
+        Self(self.0.wrapping_shr_vartime(shift))
     }
```

Comment:
@fjarri could use a little help on this one... this seems broken after updating `crypto-bigint` to v0.6.0-pre.9

Reply:
See example failures here: https://github.com/RustCrypto/elliptic-curves/actions/runs/7480942676/job/20361467871

Reply:
This seems to be a legitimate bug. Going to try to fix it upstream.

Reply:
Fixed https://github.com/RustCrypto/crypto-bigint/pull/552